cpp/src/backends/process/model_worker.cc

```diff
@@ -73,17 +75,18 @@ namespace torchserve {
       sock_addr.sin_port = port_;
       sock_addr.sin_addr.s_addr = inet_addr(socket_name_.c_str());
       srv_sock_address = reinterpret_cast<sockaddr*>(&sock_addr);
+      name_len = sizeof(*srv_sock_address);
```

Comment:
curious why we are using size of `srv_sock_address` here and size of `sock_addr` in unix path? does it matter?

Reply:
In case of `uds` casting to `sockaddr` and using `sizeof(*srv_sock_address)` was truncating the file name to 14 chars. This was not a problem with `tcp`